concepts/Common/Security Considerations/30 Export Vulnerabilities/05 CSV Injection.md

```diff
@@ -0,0 +1,191 @@
+If you use DataGrid or PivotGrid Export functionality, you should consider a threat of a CSV Injection Attack (also called formula injection attack). It involves the injection of harmful characters that act as a formula and are executed within a computer network. Cell values that begin with the **=, +, -** and **@** characters can initiate an injection attack.
+
+When you enter the malicious code into CSV files (in the form of a formula), the formula can be executed. Once executed, the malicious formula can manipulate/delete user data or provide unauthorized access to data/internal resources.
```

Comment:
> When you enter **the** malicious code into CSV files (in the form of a formula)

Normally, "code" is uncountable and requires no article unless we have a very good reason for placing an extra emphasis on this word.

> When you enter the malicious code into CSV files (in the form of a formula)

It's strange that we imply that our reader places malicious code into CSV files, moreover, intentionally... Moreover, this sentence repeats the very thing we said in the first paragraph. We can safely delete this sentence and lose nothing.

> manipulate

Normally, we do not use this word in documentation due to its negative connotation. In this case, negative connotation is justified, however, I believe there is a better alternative. The word "tamper", for example, is tightly associated with malicious code and its activity.

```suggestion
When executed, malicious code in a formula can tamper with user data or provide unauthorized access to data and internal resources.
```

P.S. You may consider merging this sentence with the previous paragraph.